dockerfiles/ci/buster/php-8.0/Dockerfile

```diff
@@ -79,10 +89,13 @@ RUN set -eux; \
         pecl install xdebug-3.0.0; \
         cd $(php-config --extension-dir); \
         mv xdebug.so xdebug-3.0.0.so; \
+        if [[ ${phpVer} != *asan* ]]; then echo "zend_extension=opcache.so" >> ${iniDir}/../php-apache2handler.ini; fi; \
```

Comment:
Does opcache fail asan?

Reply:
Not, this does no harm. I had moved the check to switch-php, because you cannot load the asan build with apache...
But I forgot to include opcache in all but the PHP 8.0 builds.